src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Native lazy loading for the web
+subhead: Browser-level native lazy loading is finally here!
+authors:
+  - houssein
+  - addyosmani
+  - mathiasbynens
+date: 2019-06-20
+hero: hero.png
+alt: Phone outline with loading image and assets
+description: |
+  This post covers the loading attribute and how it can be used
+  to defer the loading of images and iframes
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+Support for natively lazy-loading images and iframes is coming to the web!
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="./lazyload.webm" type="video/webm">
+    <source src="./lazyload.mp4" type="video/mp4">
+  </video>
+ <figcaption class="w-figcaption w-figcaption--fullbleed">
+    <a href="https://mathiasbynens.be/demo/img-loading-lazy">Lazy loading demo</a>
```

Comment:
There's a few demos here (this one and the firebaseapp one below). Do we want to rehost them on web.dev somehow?

Reply:
I personally don't mind if they're hosted separately for the meantime, but that's a good point.

@robdodson have we been making sure all demos are hosted on web.dev?

Reply:
My concern was not about remixability(tm) but rather about making sure the content stays around forever. If we link to external websites (like Glitch or my own website) there's no guarantee the content will stay up.

To be clear, I'm okay with landing this as-is, and definitely don't want to block this PR :) Longer-term, I think it would be nice to self-host demos, or at least have some kind of backup strategy in case Glitch / my website / any other places we point to for demos ever shut down.